contracts/examples/DXswapRelayer.sol

```diff
@@ -260,11 +254,10 @@ contract DXswapRelayer {
 
         if (tokenA == address(0)) {
             TransferHelper.safeTransferETH(owner, amountA);
```

Comment:
Here if the address is zero, dont you have the tokenA balance in WETH instead of ETH? because you are transferring ETH.